### ISSUE TITLE
The skip-link target should exist and be focusable

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#18]
Language: English

Tool(s): aXe

Browser/Screen reader: Firefox

WCAG 2.0 reference: Deque best practice

Details: aXe reported that : The skip-link target should exist and be focusable
Issue description: Ensure all skip links have a focusable target
Why it matters: Screen readers announce content sequentially as it appears in the HTML file. What this means for users of assistive technology is that the content at the top of the page, typically including the entire navigation, is read out to the user before reaching any of the main content. Since content at the top of the page can often be very lengthy, it can be time-consuming to listen to or tab through all of it when the user is only interested in the main content. Including a skip link in an HTML page is beneficial to blind users, users with low vision, and mouse-only users.

Element location:
#wb-auto-5
Element location:
<a class="sitemap wb-lbx wb-init wb-lbx-inited" title="Sitemap" data-effect="mfp-zoom-in" href="#sitemap_pop" id="wb-auto-5">Sitemap</a>

To solve this violation, you need to:
Fix the following: No skip link target
